CONTRIBUTING.md

```diff
@@ -33,7 +34,34 @@ The following text equips you with knowledge that makes contributing to ilastik
 
    ```
    conda config --set channel_priority strict
-   conda install --name base -c conda-forge mamba
+   conda install mamba -n base -c conda-forge
+   ```
+
+   If [strict channel priority][strict-channel-priority] does not allow installation of [Mamba][mamba]:
+
+   ```
+   conda config --set channel_priority false
+   conda install mamba -n base -c conda-forge
+   ```
+
+1. Initialize [Mamba][mamba]:
+
+   ```
+   mamba init
+   ```
+
+   If using another shell than bash such as zsh, manually add to the .zshrc file after `unset __conda_setup`:
+
+   ```shell
+   if [ -f "/opt/conda/etc/profile.d/mamba.sh" ]; then
+      . "/opt/conda/etc/profile.d/mamba.sh"
+   fi
+   ```
+
+   Restart shell or source:
+
+   ```shell
+   source ~/.zshrc
```

Comment:
This is a very peculiar case of someone having done an installation of conda-/mini-/mambaforge without adding it to their profile. While this fix is useful, I think it is too much of a special case to be included in the contributing guidelines.